projects/aws/eks-anywhere-packages/docker/linux/ecrtokenrefresher/Dockerfile

```diff
@@ -0,0 +1,12 @@
+ARG BASE_IMAGE # https://gallery.ecr.aws/eks-distro-build-tooling/eks-distro-minimal-base
+FROM $BASE_IMAGE
+
+ARG TARGETARCH
+ARG TARGETOS
+
+COPY _output/bin/eks-anywhere-packages/$TARGETOS-$TARGETARCH/ecrtokenrefresher /ecr-refresh
+COPY _output/LICENSES /LICENSES
+COPY ATTRIBUTION.txt /ATTRIBUTION.txt
```

Comment:
This will be the other attribute file name

Reply:
Updated